var/spack/repos/builtin/packages/perl-xml-libxslt/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PerlXmlLibxslt(PerlPackage):
+    """Interface to the GNOME libxslt library."""
+
+    homepage = "https://metacpan.org/pod/XML::LibXSLT"
+    url      = "https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXSLT-1.96.tar.gz"
+
+    version('1.96', sha256='2a5e374edaa2e9f9d26b432265bfea9b4bb7a94c9fbfef9047b298fce844d473')
+
+    depends_on('libxslt')
```

Comment:
It looks like this package may have more dependencies: https://cpandeps.grinnz.com/?dist=XML-LibXSLT&phase=build&perl_version=v5.30.0&style=auto

Reply:
Yes, you are correct. I checked and it did complain at build time but still proceeded.
Also thanks for the pointer to that tool. I've pushed the fix.